crates/configuration/src/parachain.rs

```diff
@@ -856,6 +864,38 @@ impl<C: Context> ParachainConfigBuilder<WithId, C> {
             ),
         }
     }
+
+    pub fn with_collator_group(
+        self,
+        f: impl FnOnce(GroupNodeConfigBuilder<node::Initial>) -> GroupNodeConfigBuilder<node::Buildable>,
+    ) -> Self {
```

Comment:
We should check if the node_group is >= 1 and then return a `buildable ` instance `

```suggestion
    ) -> ParachainConfigBuilder<WithAtLeastOneCollator, C> {
```

Reply:
Oks, sounds good 👍